reward_kit/cli.py

```diff
@@ -217,6 +218,48 @@ def parse_args(args=None):
         "If not specified, defaults to value in rewardkit.yaml or 'api-key'. Optional.",
     )
 
+    # Deploy MCP command
+    deploy_mcp_parser = subparsers.add_parser(
+        "deploy-mcp", help="Deploy an MCP server to Google Cloud Run"
```

Comment:
Out of curiosity, did you look into Cloud Run Functions? If the server does not need to be stateful, Functions may be work considering.